### PR TITLE
Microsoft.NETCore.App 1.0.0 => 1.1.0

### PR DESCRIPTION
**Pull request by troydai**
@pranavkm @javiercn

### CODE REVIEW

File: src/SplitPackages/Program.cs [hunk 1]
```diff
@@ -227,7 +227,7 @@ private ProjectJsonFileBuilder CreateBaseJsonFileBuilderForCache(string destinat
 
             var hardcodedDependencies = new[]
             {
-                new PackageInformation(null, "Microsoft.NetCore.App", "1.0.0", new[] { Frameworks.NetCoreApp10 })
+                new PackageInformation(null, "Microsoft.NetCore.App", "1.1.0", new[] { Frameworks.NetCoreApp10 })
```
Comment (pranavkm) [inline src/SplitPackages/Program.cs]: Would this affect tools? We're still trying to figure out if tools should continue targeting `Microsoft.NETCore.App 1.0.0`.

Comment (troydai) [inline src/SplitPackages/Program.cs]: This is used to generate the cache.project.json and final.cache.project.json file. These two files are used to pick correct packages for package cache.

Are these two files used for other purpose? Or is there other unexpected outcome of this tool?

/cc @javiercn

Comment (pranavkm) [inline src/SplitPackages/Program.cs]: I wasn't sure if having a `netcoreapp1.0` targeting tool would affect this (those are the only things we ship that target netcoreapp). If this is specifically used to produce a project.json, this should be fine

Comment (troydai) [inline src/SplitPackages/Program.cs]: from my reading on the source code, this is the case.

**Comment by pranavkm** [comment]
:shipit:

**Pull request merged**